src/Grpc/test/testassets/InteropClient/InteropClient.csproj

```diff
@@ -1,11 +1,9 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <OutputType>Exe</OutputType>
-    <TargetFramework>netcoreapp5.0</TargetFramework>
+    <TargetFramework>$(DefaultNetCoreTargetFramework)</TargetFramework>
```

Comment:
I made the interop client and website separate from the overall build infrastructure because they needed to reference the installed ASP.NET SDK, and not ASP.NET Core projects.

It appears you're integrating them into the overall build. How did you solve the ASP.NET SDK referencing issue?

Reply:
By ASP.NET SDK do you mean the shared framework? So far I haven't done that yet, I'm only changing how the packages are referenced.

Though I see why we need to reference the shared framework in the InteropWebsite, why is it needed for the InteropClient? It seems to me that it should really just be a regular .NET Core app.